### ISSUE TITLE
Payment Request API & Apple Pay

### ISSUE THREAD

**Issue by jackbrycesmith** [flaviocopes/grav-plugin-shoppingcart-stripe#4]
Hi Flavio

Would you be open to adding support for checkout completion via Payment Request API & Apple Pay to this plugin?

Here's the woocommerce implementation:
- https://github.com/woocommerce/woocommerce-gateway-stripe
- Demo: https://woocommerce.paymentrequest.show/

I haven't been able to try Apple Pay, and understand it would involve additional configuration steps vs Payment Request API (i.e. you have to (1) enable it on your Stripe account & (2) setup a file in a `/.well-known/` directory in your server root).

**Comment by deggenschwiler** [comment]
@jackbrycesmith did you ever get this working - I'm looking for the same solution.

**Comment by jackbrycesmith** [comment]
@deggenschwiler I did get a hacky version working at the time but didn't pursue it seriously as it would have involved extra PCI compliance burdens. 

~~Using the new [stripe.js v3 payment request element](https://stripe.com/docs/elements/payment-request-button) would save you a lot of the work, plus it would solve the PCI compliance concern. I found it straightforward to implement in the [Proposal Plugin](https://github.com/jackbrycesmith/grav-plugin-proposal).~~ 

Ahh just seen [your fork](https://github.com/deggenschwiler/grav-plugin-shoppingcart-stripe), will keep an eye out for how you get on.
  
Closing this for now as I think it would make more sense to do in a separate plugin.
